common/scripts/testAll.sh

```diff
@@ -3,11 +3,13 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
This file is IMO also redundant (user should just run `./gradlew test` in the root directory)

Reply:
Just FTR, `./gradlew test` runs _unit tests_. `./gradlew functionalTest` runs functional tests. `./gradlew fullFunctionalTest` runs functional tests, for all supported Gradle versions, for Gradle. And `./gradlew check` runs all verifications (tests, checkstyle, ...).

Reply:
In that case this script wasn't even working correctly (`testAll` is a wrong name since it runs only a single gradle and maven test).